static/src/javascripts/projects/commercial/modules/dfp/define-slot.js

```diff
@@ -26,12 +30,18 @@ const buildSizeMapping = (sizes) => {
 	breakpoints
 		.filter((_) => _.name in sizes)
 		.forEach((_) => {
-			mapping.addSize([_.width, 0], sizes[_.name]);
+			mapping.addSize([_.width, 0], sizes[_.name].map((size) => ((!size[0] && !size[1]) ? 'fluid' : size)));
```

Comment:
Anyway we could improve the readability here a bit?

```suggestion
			mapping.addSize([_.width, 0], sizes[_.name].map((size) => ((size[0] === 0 && size[1]) === 0 ? 'fluid' : size)));
```

or

```suggestion
			mapping.addSize([_.width, 0], sizes[_.name].map((size) => ((isFluid(size) ? 'fluid' : size)));
```